### DISCUSSION TITLE
Types aliases? (Domains)

### DISCUSSION THREAD

**Discussion by Zer0xxxx** [alfateam/rdb#97] [Q&A]
Hello!
First of all, thank you a lot for this fp ORM.

I would like to know how to use type aliases with Orange ORM?
I don't see any clue about it in the doc.

I would like to use some types aliases, like so:
```sql
CREATE DOMAIN SUPPLIER_ID AS _;
CREATE DOMAIN ACCOUNT_ID AS _;
CREATE DOMAIN ACCOUNT_HISTORY_ID AS _;
```

**Comment by lroal** [comment]
I am not sure where you would benefit from it today - as Orange does not produce any migrations.
But maybe I don't see the context here..

**Reply by Zer0xxxx** [reply]
Perhaps I haven't yet understood exactly what Orange ORM does and doesn't do. It's just that I'd have liked to be able to define syntactic sugars over `.numeric()` or that sort of functions. Just to know a little better what I'm handling when I read again the code, without it changing absolutely nothing under the hood.
